src/epr/main.py

```diff
@@ -0,0 +1,554 @@
+            dest="name",
+            action="store",
+            default=None,
+            help="Name of the Event Receiver",
+        )
+        event_receiver_parser.add_argument(
+            "--type",
+            dest="type",
+            action="store",
+            default=None,
+            help="Type of the Event Receiver",
+        )
+        event_receiver_parser.add_argument(
+            "--version",
+            dest="version",
+            action="store",
+            default=None,
+            help="Version of the Event Receiver",
+        )
+        event_receiver_parser.add_argument(
+            "--description",
+            dest="description",
+            action="store",
+            default=None,
+            help="Description of the Event Receiver",
+        )
+        event_receiver_parser.add_argument(
+            "--schema",
+            dest="schema",
+            action="store",
+            default=None,
+            help="Schema of the Event Receiver",
+        )
+        event_receiver_parser.add_argument(
+            "--fields",
+            dest="fields",
+            action="store",
+            default=None,
+            help="Fields of the Event Receiver",
+        )
+        event_receiver_group_parser = subparsers.add_parser(
+            "event-receiver-group", help="Event Receiver Group related options"
+        )
+        event_receiver_group_parser.add_argument(
+            "--id",
+            dest="id",
+            action="store",
+            default=None,
+            help="ID of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--name",
+            dest="name",
+            action="store",
+            default=None,
+            help="Name of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--type",
+            dest="type",
+            action="store",
+            default=None,
+            help="Type of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--version",
+            dest="version",
+            action="store",
+            default=None,
+            help="Version of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--description",
+            dest="description",
+            action="store",
+            default=None,
+            help="Description of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--event-receiver-ids",
+            dest="event_receiver_ids",
+            action="store",
+            default=None,
+            help="Event Receiver IDs of the Event Receiver Group",
+        )
+        event_receiver_group_parser.add_argument(
+            "--fields",
+            dest="fields",
+            action="store",
+            default=None,
+            help="Fields of the Event Receiver Group",
```

Comment:
Should mention comma separated/delimited. Applies to `--fields` flag for other commands as well